packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/utils.ts

```diff
@@ -89,3 +92,13 @@ export function getSourceMapDevTool(
     append: hiddenSourceMap ? false : undefined,
   });
 }
+
+export function getEsVersionForFileName(
```

Comment:
Nit: We don't typically use this pattern

Reply:
Which pattern do you mean?

Reply:
```
  params: {
    scriptTargetOverride: typescript.ScriptTarget | undefined,
    esVersionInFileName: boolean,
  }): string {
```